sdk/extensions/wcf/service.projects

```diff
@@ -0,0 +1,24 @@
+<!--
+  This file allows overrides and extensions to the build configuration defined
+  by the central engineering system, and is considered during pipeline builds for
+  CI, test runs, and package release.
+
+  Based on the SDKType variable (msbuild property or env variable) this will filter
+  the set of projects to to match what type of SDK we are interested in building.
+
+  SDKType values
+   - "all" will build all projects
+   - "source" will build the source code for WCF
+   - "test" will build the test projects
+-->
+
+<Project>
+ 
+  <ItemGroup Condition="'$(SDKType)' == 'source'">
```

Comment:
This is not a pattern that we generally use in the repository.   Can you help me understand the intent?

Reply:
Are you talking about the source/test aspect in this folder? I'm not sure, I didn't create this file. I think @subhra231 got confused about the purpose of this file. It looks like it shouldn't be there. We do need a service.projects file in the parent extensions folder to prevent ci.yml from building the WCF source. I presume you aren't talking about using service.projects files in general as they are all through the repository.

Reply:
Not in general, nor the one in the /sdk/extensions root, which we need.    Just this specific one.